index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Ben Johnson's Portfolio</title>
+    <link rel='stylesheet' href='css/normalize.css'>
+    <link rel='stylesheet' href='css/style.css'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap" rel="stylesheet">
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+  </head>
+  <body>
+    <!-- button for scrolling to top of page -->
+    <button onclick="topFunction()" id="upbutton" title="Go to top">Top</button>
+    <script src='js/upbutton.js'></script>
+    <header class='header'>
+      <h1>Ben Johnson<span class='blink-text'>_</span></h1>
+    </header>
+    <nav class='nav'>
+      <button type="button" name="button" onclick="location.href='#about-section'">About</button>
+      <button type="button" name="button" onclick="location.href='#project-section'">Projects</button>
+      <button type="button" name="button" onclick="window.open('https://github.com/bnjn', '_blank');">GitHub</button>
+      <button type="button" name="button" onclick="location.href='docs/cv.pdf'">Download CV</button>
+    </nav>
+    <section id='about-section' class='about'>
+      <h2>About</h2>
+      <div class='about-flexbox'>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed scelerisque nulla et velit elementum, ut pretium enim pellentesque. Ut pretium venenatis tempus. Integer varius tempus feugiat. Vivamus a imperdiet risus. Fusce eu tincidunt purus. Ut bibendum sapien eros, ut viverra risus semper ac. Fusce fringilla metus at quam dignissim, a fermentum lacus lacinia. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer elementum lorem vel sapien iaculis eleifend. Nunc ullamcorper facilisis risus ac luctus. Nam finibus turpis nibh, vel interdum arcu lacinia vitae. Sed non nunc sapien. Donec cursus orci ligula, nec porttitor mauris feugiat eu. Aenean tellus lorem, iaculis in sodales quis, tincidunt lobortis purus. Sed non odio quis nunc rutrum pellentesque vitae euismod metus. Sed aliquam mauris sit amet ex aliquet porttitor.</p>
+        <img src='img/profile-photo.png' alt='photo of ben johnson'>
+      </div>
+    </section>
+    <section id='project-section' class='projects'>
+      <h2>Projects</h2>
+      <div class='project-container'>
+        <div>
+          <div class='button-overlay'>
+            <button type="button" name="button" class='about-button'>About</button>
+            <button type="button" name="button" onclick="window.open('https://github.com', '_blank');">GitHub</button>
+          </div>
+          <img src='img/placeholder.png' alt='cat'>
+        </div>
```

Comment:
Could be 1 indentation to the right to join up with the div element above it perhaps.

Reply:
Resolved in person.